inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -425,11 +426,13 @@ def refextract(obj, eng):
         if tmp_document:
             pdf_references = extract_references_from_pdf(tmp_document, source)
             pdf_references = [match_reference(ref) for ref in pdf_references]
+            pdf_references = dedupe_list(pdf_references)
```

Comment:
This is not tested. I know it is not easy, so I am fine with skipping this test for now, but please create a US to point to this comment. I'll do it myself actually.

Reply:
https://its.cern.ch/jira/browse/INSPIR-652